spec/services/section/create_service_spec.rb

```diff
@@ -0,0 +1,96 @@
+require "spec_helper"
+
+RSpec.describe Section::CreateService do
+  let(:user) { User.gds_editor }
+  let(:manual) { Manual.new(title: 'manual-title') }
+  let(:section_attributes) { double(:section_attributes) }
```

Comment:
I agree that it's not ideal. I ended up going with a double so that I could be confident that the `attributes` passed to `Section::CreateService#initialize` was the same as the attributes passed to `Manual#build_section`.
I'm going to leave this as it is at the moment and see whether I come up with anything better while working on the rest of the services.